### DISCUSSION TITLE
Even simpler VUEngine helloworld FYI

### DISCUSSION THREAD

**Discussion by loociano** [VUEngine/VUEngine-Barebone#4] [Show and tell]
Hi @jorgeche @KR155E,

Thanks for creating VUEgine Studio!

FYI I forked this sample project to create an even simpler `helloworld` program in VUEngine, without plugins, animations, intro, language selection or automatic pause screens. https://github.com/loociano/VUEngine-helloworld/ Basically the dryest helloworld I could think of.

**Comment by KR155E** [comment]
Hey @loociano,

I enabled discussions in this repo and converted over your issue so we can preserve it in a more fitting format.

That's pretty cool! I think it's the first ever release of a "third party" VUEngine project. Hope it will be useful for other new VUEngine developers.

We are looking forward to what might come next. :-)
